### PR TITLE
add issue template and support md

### PR DESCRIPTION
**Pull request by cldougl**
📝 cc @Marc-Andre-Rivet  @chriddyp @valentijnnieman

### CODE REVIEW

**Comment by valentijnnieman** [comment]
Looks great! While we're at it, perhaps now is a good time to add a contribution template as well? Would be handy to have, maybe it could include asking for a test and changelog + version update? 

What about a code of conduct? This https://www.contributor-covenant.org/ could help us out perhaps.

**Comment by chriddyp** [comment]
> While we're at it, perhaps now is a good time to add a contribution template as well? Would be handy to have, maybe it could include asking for a test and changelog + version update?

Yeah good idea. Let's try to keep things standard across repos. Here is the ones we have from Dash: https://github.com/plotly/dash/blob/master/CONTRIBUTING.md, https://github.com/plotly/dash/blob/master/CODE_OF_CONDUCT.md

**Comment by cldougl** [comment]
We have a CONTRIBUTING.md already in this repo: https://github.com/plotly/dash-table/blob/master/CONTRIBUTING.md

@valentijnnieman @chriddyp are you suggesting I edit the current CONTRIBUTING.md to make it more consistent with Dash repo or that I add a PULL_REQUEST_TEMPLATE.md in this PR?

I'll add the CODE_OF_CONDUCT.md now.

**Comment by chriddyp** [comment]
> are you suggesting I edit the current CONTRIBUTING.md to make it more consistent with Dash repo or that I add a PULL_REQUEST_TEMPLATE.md in this PR?

Right, yeah, good point. I'd say let's keep everything centralized in `CONTRIBUTING.md` but add things to the bottom.

Almost everything in the Dash `CONTRIBUTING.md` is fair game. The only difference is how pre-release versions happen - that's documented pretty well in the `dash-core-components` `CONTRIBUTING.md`: https://github.com/plotly/dash-core-components/blob/master/CONTRIBUTING.md#publishing-new-componentsfeatures.

**Comment by cldougl** [comment]
@Marc-Andre-Rivet @valentijnnieman can you guys double check the rc instructions- specifically versioning (want to make sure I got that correct as afaik there's no `version.py` at this time.)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,29 +1,42 @@
-## Setup
+# Contributing to Dash Table
+
+## Getting Started
+Refer to the [readme](README.md) for installation and basic use instructions.
+
+## Coding Style
+Please lint any additions to Python code with `pylint` and `flake8`.
```
Comment (Marc-Andre-Rivet) [inline CONTRIBUTING.md]: Javascript / Typsescript is linted through `npm run lint`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -33,3 +46,65 @@ Use the review apps to verify callback functionality (note these examples are wr
...
 
 Note: Distributable file will be located in ./dist
+
+## Making a Contribution
+_For larger features, your contribution will have a higher likelihood of getting merged if you create an issue to discuss the changes that you'd like to make before you create a pull request._
+
+1. Create a pull request and tag the Plotly team (`@plotly/dash`) and tag / request review from [@Marc-Andre-Rivet](https://github.com/Marc-Andre-Rivet) and [@valentijnnieman ](https://github.com/valentijnnieman).
+2. After a review has been done and your changes have been approved, create a prerelease and comment in the PR. Version numbers should follow [semantic versioning][]. To create a prerelease:
+    * Add `-rc1` to `package.json` e.g. `0.14.0-rc1`
+    * Update the `unpkg` link in `./dash_table/__init__.py`, replacing `__version__` with your release candidate (e.g. `"0.14.0-rc1"`)
```
Comment (Marc-Andre-Rivet) [inline CONTRIBUTING.md]: I think this is wrong. If you npm run build:js && npm run build:py, shouldn't the __init__.py pick up the new version automatically?

**Pull request merged**